### DISCUSSION TITLE
Using Gitlab Renovate runner without custom Gitlab runner

### DISCUSSION THREAD

**Discussion by felixranesberger** [renovatebot/renovate#20671] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Renovate Runner v10.86.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

Gitlab CE

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate currently throws an error, when using it with our Gitlab CI. 

`unable to resolve docker endpoint: open /certs/client/ca.pem: no such file or directory`

I found[ this readme](https://github.com/renovatebot/docker-renovate/blob/HEAD/docs/gitlab.md#gitlab-runner-config), where it tells me to create a custom Gitlab runner for running Renovate. 

Is it possible to run Renovate without a custom runner and without adjusting our current runner?

![Screenshot 2023-02-28 at 15 29 40](https://user-images.githubusercontent.com/52704891/221883653-1aa85b3f-4642-4c32-a092-75db7fa13151.png)

### What you tried so far.

/

### Relevant debug logs

```
WARN: Error removing dangling containers
"err": {
    "name": "ExecError",
    "cmd": "/bin/sh -c docker ps --filter label=renovate_child -aq",
    "stderr": "unable to resolve docker endpoint: open /certs/client/ca.pem: no such file or directory\n",
    "stdout": "",
    "options": {"encoding": "utf-8"},
    "exitCode": 1,
    "message": "Command failed: docker ps --filter label=renovate_child -aq\nunable to resolve docker endpoint: open /certs/client/ca.pem: no such file or directory\n",
    "stack": "ExecError: Command failed: docker ps --filter label=renovate_child -aq\nunable to resolve docker endpoint: open /certs/client/ca.pem: no such file or directory\n\n    at ChildProcess.<anonymous> (/opt/buildpack/tools/renovate/34.80.0/node_modules/renovate/lib/util/exec/common.ts:99:11)\n    at ChildProcess.emit (node:events:513:28)\n    at ChildProcess.emit (node:domain:489:12)\n    at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)"
}
```

**Comment by viceice** [comment, marked as answer]
set binary source to install via ENV

**Reply by felixranesberger** [reply]
Worked like a charm! Thanks so much for all the help, you gave when setting up Renovate. 
You guys are awesome!

**Reply by viceice** [reply]
i think I'll switch the dind templates to install mode soon, or just add template for install mode.
